Source/Schema.NET/OneOrMany{T}.cs

```diff
@@ -107,7 +107,7 @@ public int Count
         /// <param name="item">The single item value.</param>
         /// <returns>The result of the conversion.</returns>
 #pragma warning disable CA2225 // Operator overloads have named alternates
-        public static implicit operator OneOrMany<T>(T item) => new OneOrMany<T>(item);
+        public static implicit operator OneOrMany<T>(T item) => item != null && item.GetType() == typeof(string) && string.IsNullOrWhiteSpace(item as string) ? default : new OneOrMany<T>(item);

```

Comment:
If you know it's a string, you can just cast it instead of `item as string`.

Reply:
Apparently not with generics, if I switch to `(string)item` I get a compiler error of `Cannot convert type 'T' to 'string'`